scripts/core/helpers/utils.tsx

```diff
@@ -31,6 +31,10 @@ export function pick<T extends IOnlyStringKeys, K extends keyof T>(obj: T, ...ke
     return picked;
 }
 
+export function getObjectEntriesGeneric<KEY extends string, VALUE>(obj: any): Array<[KEY, VALUE]> {
+    return Object.entries(obj) as Array<[KEY, VALUE]>;
+}
+
```

Comment:
Unused code

Reply:
There's an exception to the unused code rule for utilities. I also wrote some e2e helpers(addFormattingOptionToContentProfile) that I'm not using at the moment, but will probably need later.